[2024-05-18T19:17:15.696+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-18T19:17:15.711+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_loader.hello_task scheduled__2023-04-09T01:00:00+00:00 [queued]>
[2024-05-18T19:17:15.718+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_loader.hello_task scheduled__2023-04-09T01:00:00+00:00 [queued]>
[2024-05-18T19:17:15.719+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-18T19:17:15.729+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): hello_task> on 2023-04-09 01:00:00+00:00
[2024-05-18T19:17:15.735+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1138) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-18T19:17:15.737+0000] {standard_task_runner.py:63} INFO - Started process 1140 to run task
[2024-05-18T19:17:15.737+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_loader', 'hello_task', 'scheduled__2023-04-09T01:00:00+00:00', '--job-id', '257', '--raw', '--subdir', 'DAGS_FOLDER/data_loader.py', '--cfg-path', '/tmp/tmpf7vp9tmm']
[2024-05-18T19:17:15.739+0000] {standard_task_runner.py:91} INFO - Job 257: Subtask hello_task
[2024-05-18T19:17:15.774+0000] {task_command.py:426} INFO - Running <TaskInstance: data_loader.hello_task scheduled__2023-04-09T01:00:00+00:00 [running]> on host b5cbc22da81e
[2024-05-18T19:17:15.837+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_loader' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2023-04-09T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-09T01:00:00+00:00'
[2024-05-18T19:17:15.838+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-18T19:17:15.839+0000] {logging_mixin.py:188} INFO - 2023-04-09/
[2024-05-18T19:17:15.840+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-18T19:17:15.840+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-18T19:17:15.848+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_loader, task_id=hello_task, run_id=scheduled__2023-04-09T01:00:00+00:00, execution_date=20230409T010000, start_date=20240518T191715, end_date=20240518T191715
[2024-05-18T19:17:15.871+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-18T19:17:15.885+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-18T19:17:15.888+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
